swatch-system-conduit/src/main/spec/internal-organizations-sync-api-spec.yaml

```diff
@@ -0,0 +1,314 @@
+openapi: "3.0.2"
+info:
+  title: "internal-organizations-sync-api"
+  description: "REST interface for the internal-organization-sync service. Please note any deprecated APIs. Our current deprecation policy is to keep deprecated APIs around for at least 6 months."
+  version: 1.0.0
+paths:
+  /internal/rpc/syncOrg:
+    description: "Trigger a sync for a given Org ID"
+    post:
+      summary: "Sync organization for given org_id."
+      requestBody:
+        $ref: "#/components/requestBodies/OrgSyncRequestBody"
+      operationId: syncOrg
+      tags:
+        - internalOrganizations
+      responses:
+        '200':
+          description: "The request for syncing organization has completed successfully."
+          content:
+            application/vnd.api+json:
+              schema:
+                $ref: "#/components/schemas/DefaultResponse"
+        '400':
+          $ref: "../../../../spec/error-responses.yaml#/$defs/BadRequest"
+        '403':
+          $ref: "../../../../spec/error-responses.yaml#/$defs/Forbidden"
+        '404':
+          $ref: "../../../../spec/error-responses.yaml#/$defs/ResourceNotFound"
+        '500':
+          $ref: "../../../../spec/error-responses.yaml#/$defs/InternalServerError"
+  /internal/rpc/syncAllOrgs:
+    description: "Trigger a sync for all Organizations"
+    post:
+      summary: "Sync all organizations."
+      operationId: syncFullOrgList
+      tags:
+        - internalOrganizations
+      responses:
+        '202':
+          description: "The request for syncing all organizations is successfully running."
+          content:
+            application/vnd.api+json:
+              schema:
+                $ref: "#/components/schemas/DefaultResponse"
+        '400':
+          $ref: "../../../../spec/error-responses.yaml#/$defs/BadRequest"
+        '403':
+          $ref: "../../../../spec/error-responses.yaml#/$defs/Forbidden"
+        '404':
+          $ref: "../../../../spec/error-responses.yaml#/$defs/ResourceNotFound"
```

Comment:
Would this call actually return a 404, since it's not a call to a specific org but a general global operation?